provider/scripts/spark/offline_store_spark_runner.py

```diff
@@ -71,8 +71,11 @@ def display_data_metrics(df, spark):
     if len(rows_as_list) > 100:
         rows_as_list = rows_as_list[:100]
     for row in rows_as_list:
-        row_list = row.asDict().values()
-        rows.append(list(row_list))
+        row_values = list(row.asDict().values())
+        for i, curr_value in enumerate(row_values):
+            if isinstance(curr_value, datetime.datetime):
+                row_values[i] = curr_value.strftime("%m/%d/%Y, %H:%M:%S")
```

Comment:
wondering if this makes more sense to use a dataframe method here instead of iterating through everything